library/src/main/java/com/ToxicBakery/viewpager/transforms/CubeOutTransformer.java

```diff
@@ -22,6 +22,7 @@
 
 	@Override
 	protected void onTransform(View view, float position) {
+		view.setCameraDistance(view.getWidth()*20);
```

Comment:
Can you explain why the value of 20? Also the formatting is off as this should be `view.getWidth() * 20`

Reply:
actually there is no reason for using 20. you can use any value but if you change it there will be a difference in appearance of transformation. in was opinion it was better looking with value of 20.
we can get this value in the constructor of this class if you want to.

Reply:
Yeah, add it as an overloaded constructor and you can leave 20 as the default for the no-arg constructor.

Reply:
i figured out that you are not using any constructors! i can pass the the parameters with .getConstructor() method but it will make your sample app confusing.

Reply:
That's why I was saying to use overloaded constructors.

```java
public CubeOutTransformer() {
  this(20);
}

public CubeOutTransformer(int distanceMultiplier) {
  this.distanceMultiplier = distanceMultiplier;
}
```

Reply:
done